packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/studio/services/StudioProjectTemplateInitializer.java

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+ * Copyright (c) 2024, 2024 Obeo.
```

Comment:
Nitpick: rhere are several instances of "2024, 2024" in different files.

Reply:
I've fixed a dozens or so of those, thanks 👍